FsAutoComplete/CompilerServiceInterface.fs

```diff
@@ -0,0 +1,159 @@
+namespace FsAutoComplete
+
+open System
+open System.IO
+open Microsoft.FSharp.Compiler
+open Microsoft.FSharp.Compiler.SourceCodeServices
+
+type ParseAndCheckResults(parse: FSharpParseFileResults,
+                          check: FSharpCheckFileResults,
+                          version: int) =
+  let parseResults = parse
+  let checkResults = check
+  let _version = version
```

Comment:
The `_version` variable seems unused - and I don't think you need new let bindings for `parseResults` and `checkResults` either - could you just rename the constructor arguments?

(Or am I missing something? They are not mutable so it should be fine to use the compiler parameters...)